webpack/base.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     ),
     vendors: path.resolve(__dirname, '../commcare_connect/static/js/vendors'),
     tailwind: path.resolve(__dirname, '../tailwind/tailwind.css'),
+    mapbox: path.resolve(__dirname, '../commcare_connect/static/js/mapbox'),
```

Comment:
does this load mapbox js on every page? if so, do we want that?

Reply:
No, this is just the command for building it, it's only included where it's needed. For e.g. right now in [user_visit_verification.html](https://github.com/dimagi/commcare-connect/pull/571/files#diff-28499778226712ea99486793c595dd7eb1ec1a21d9f130db8ff9d03fd4f24149R14)